java/connector-framework-internal/src/main/java/org/identityconnectors/framework/impl/serializer/CommonObjectHandlers.java

```diff
@@ -475,8 +455,7 @@ public Object deserialize(final ObjectDecoder decoder) {
                 final boolean container = decoder.readBooleanField("container", false);
                 final boolean auxiliary = decoder.readBooleanField("auxiliary", false);
 
-                @SuppressWarnings("unchecked")
-                final Set<AttributeInfo> attrInfo =
+                @SuppressWarnings("unchecked") final Set<AttributeInfo> attrInfo =

```

Comment:
Please avoid such formatting-only changes.

Reply:
Thank you, Ill fix this one as well.